PhysicsTools/JetMCAlgos/python/AK5PFJetsMCFlavourInfos_cfi.py

```diff
@@ -4,7 +4,7 @@
     jets                     = cms.InputTag("ak5PFJets"),
     bHadrons                 = cms.InputTag("selectedHadronsAndPartons","bHadrons"),
     cHadrons                 = cms.InputTag("selectedHadronsAndPartons","cHadrons"),
-    partons                  = cms.InputTag("selectedHadronsAndPartons","physicsPartons"),
+    partons                  = cms.InputTag("selectedHadronsAndPartons","algorithmicPartons"),
     leptons                  = cms.InputTag("selectedHadronsAndPartons","leptons"),
     jetAlgorithm             = cms.string("AntiKt"),
     rParam                   = cms.double(0.5),
```

Comment:
Can you also set `hadronFlavourHasPriority` to `False` here and in AK4PFJetsMCFlavourInfos_cfi.py and propagate these changes to #11077? Thanks.